pyseir/inference/model_fitter.py

```diff
@@ -210,9 +204,23 @@ def set_inference_parameters(self):
         """
         self.fit_params = self.DEFAULT_FIT_PARAMS
         # Update any state specific params.
-        for k, v in self.PARAM_SETS.items():
-            if self.state_obj.abbr in k:
-                self.fit_params.update(v)
+        initial_params_df = pd.read_csv("./pyseir/inference/world_according_to_pyseir.csv")
+
+        INITIAL_PARAM_SETS = [
+            "R0",
+            "t0",
+            "eps",
+            "t_break",
+            "eps2",
+            "t_delta_phases",
+            "test_fraction",
+            "hosp_fraction",
+            "log10_I_initial",
+        ]
+        if int(self.fips) in initial_params_df["fips"]:
```

Comment:
This call to int feels wrong. We handle fips everywhere else as strings, so in the pd.read_csv you should call that column as a string (zfill-ed correctly as needed).

Reply:
As in the fips for alabama is "01" not just "1".

Reply:
done